### ISSUE TITLE
migrate --provider option doesn't exist

### ISSUE THREAD

**Issue by a3020** [zachleigh/laravel-property-bag#3]
`php artisan migrate --provider="LaravelPropertyBag\ServiceProvider"`

That `--provider` option doesn't exist in 5.3. I assume it can be changed to `php artisan migrate`?

**Comment by zachleigh** [comment]
Yes, once the migration has been published, you can simply run `php artisan migrate` to migrate it.

**Comment by a3020** [comment]
Cool, I see you've updated the readme. Also, +1 on the documentation, I got everything up and running in no time!

**Comment by zachleigh** [comment]
Thanks! Glad you got it working quickly. Let me know if you run into any problems or have any ideas for improvements/additional features.

**Comment by a3020** [comment]
It's not always the case that if you perform `$object->set()` it will automatically persist. Maybe it is good to mention the documentation that `set` performs a sync with the database?

I was checking the property_bag table to see if a value was being saved. The table was empty so I was looking for a `persist` or `save` method. After looking into it I discovered I had set a value to it's default value, in which case it will be _removed_ from the database.
